### PR TITLE
Updated news info for adapter and test generator

### PR DESCRIPTION
**Pull request by OsirisTerje**

### CODE REVIEW

File: _posts/2018-11-23-NUnit3TestAdapter-3.11.2.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+NUnit3TestAdapter, version 3.11.2 is released.
+
+See [the release notes](https://github.com/nunit/docs/wiki/Adapter-Release-Notes) for what has changed.  Also see section for release 3.11 which contains the bugs and features. 
+
+[Download nuget package](https://www.nuget.org/packages/NUnit3TestAdapter/3.11.2)
```
Comment (jnm2) [inline _posts/2018-11-23-NUnit3TestAdapter-3.11.2.md]: Should match the capitalization of the brand name (NuGet).

File: _posts/2018-11-22-NUnitTestGenerator-2.1.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+NUnit Test Generator extension for Visual Studio 2017, version 2.1 is released.
```
Comment (jnm2) [inline _posts/2018-11-22-NUnitTestGenerator-2.1.md]: The comma seems to split the sentence in an awkward way. Don't think I've seen a comma separate the product name and the version while inside a sentence.

File: _posts/2018-11-23-NUnit3TestAdapter-3.11.2.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+NUnit3TestAdapter, version 3.11.2 is released.
```
Comment (jnm2) [inline _posts/2018-11-23-NUnit3TestAdapter-3.11.2.md]: Same comma thing.

File: _posts/2018-11-23-NUnit3TestAdapter-3.11.2.md (hunk 1 above)
Comment (OsirisTerje) [reply _posts/2018-11-23-NUnit3TestAdapter-3.11.2.md]: Fixed

File: _posts/2018-11-23-NUnit3TestAdapter-3.11.2.md (hunk 3 above)
Comment (OsirisTerje) [reply _posts/2018-11-23-NUnit3TestAdapter-3.11.2.md]: FIxed

**Comment by jnm2** [comment]
@OsirisTerje Sorry for the delay. I had to get rid of a splitting headache. For future reference should I merge right after approving next time if some time has gone by?

**Pull request merged**